src/components/30-organisms/table/index.js

```diff
@@ -0,0 +1,39 @@
+import { LitElement, css, unsafeCSS } from 'lit-element';
+import defineOnce from '../../../utils/define-once';
+import tableCss from './index.scss';
+
+// subcomponents that can only be used with axa-table:
```

Comment:
What does this comment mean? Axa-table is the definition and not a subcomponent that can be used withing axa-table, right? But I guess I'm just confused here. :)

Reply:
legacy.. i remove it